astroplan/scheduling.py

```diff
@@ -299,14 +463,54 @@ def _make_schedule(self, blocks):
         return new_blocks, True
 
 
-class PriorityScheduler(Scheduler):
+class ScheduleScheduler(object):
```

Comment:
Why is there a `Scheduler` and a `ScheduleScheduler`? Is there a more obvious name for `ScheduleScheduler`?